projects/plugins/boost/app/assets/src/js/stores/critical-css-status-ds.ts

```diff
@@ -0,0 +1,95 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Yeah, `ds` is for data-sync. I'm still on the fence about how to call all things related to data-sync. Because they are typed stores, not exactly a type-script type, not exactly a store (because it has things like API endpoints too). I'll ponder on this some more.